src/pages/listWorkers/ListWorkers.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { Worker } from "../worker/Worker";
+
+export const ListWorkers = () => {
+  const [workers, setWorkers] = useState([]);
```

Comment:
if its not too much problem `[workerdsData, setWorkersData]`

Reply:
done